src/xdotoolify.js

```
@@ -63,6 +63,10 @@ var _getElementAndBrowserRect = async function(page, selector) {
         var element = document.querySelector(_selector);
         var result = getElementVisibleBoundingRect(element);
       }
+
+      if (element == null || type(element) == NodeList && element.length == 0) {
```

shouldn't this throw instead of returning?